nautobot/core/views/generic.py

```diff
@@ -106,6 +106,7 @@ def get_extra_context(self, request, instance):
             (dict): Additional context data
         """
         return {
+            "object_detail_content": self.object_detail_content,
```

Comment:
Why's this needed now and not before?

Reply:
This is for cosmetic purposes for now. We need to figure out a better way to pass `object_detail_content` to Notes and Changelog views.

Without this line (Note that the previous tab headers are not rendered properly):
![Screenshot 2025-03-31 at 3 05 36 PM](https://github.com/user-attachments/assets/f4918ce9-7efb-4de7-bffe-a72642baf97c)

With this line
![Screenshot 2025-03-31 at 3 04 43 PM](https://github.com/user-attachments/assets/c63d36a3-e150-4382-9615-03feb4b257c2)